test/optimization/test_converters.py

```diff
@@ -406,7 +406,7 @@ def test_binary_to_integer(self):
         op.linear_constraint(linear, Constraint.Sense.EQ, 6, 'x0x1x2')
         conv = IntegerToBinary()
         _ = conv.encode(op)
-        result = OptimizationResult(x=[0, 1, 1, 1, 1], fval=17)
+        result = OptimizationResult(x=[0, 1, 1, 1, 1], fval=17, results=0)
         new_result = conv.decode(result)
         self.assertListEqual(new_result.x, [0, 1, 5])
         self.assertEqual(new_result.fval, 17)
```

Comment:
Could it be better to use `self.assertAlmostEqual` or `np.testing.assert_almost_equal` for floats? Just in case. Same for all float testing.

Reply:
In this test, the values should be integers (casted to float), so both assertAlmostEqual and assertEqual work.  But, the type hint is float, so it might be good to use the almost equal version as you said.